calcom/cal.com
# fix: saml update/delete only owner team

## What does this PR do?
Makes only owners be able to update or delete saml config.

Fixes # (issue)
9/10

## Type of change


## How should this be tested?

Make sure to enable SAML, and get a License Key.
Try to use fetch or curl to delete or updateConfigSaml of a team you´re not an owner of.

You should recieve a 401 unauthorized
